lib/matplotlib/axes/_axes.py

```diff
@@ -1926,6 +1930,8 @@ def make_iterable(x):
         width = make_iterable(width)
         _bottom = bottom
         bottom = make_iterable(bottom)
+        _tick_label = tick_label
```

Comment:
tick_label is not None, please
On Mar 29, 2015 11:33 PM, "Thomas A Caswell" <notifications@github.com>
wrote:

> In lib/matplotlib/axes/_axes.py
> <https://github.com/matplotlib/matplotlib/pull/4266#discussion_r27365250>:
>
> > @@ -1926,6 +1930,8 @@ def make_iterable(x):
> >          width = make_iterable(width)
> >          _bottom = bottom
> >          bottom = make_iterable(bottom)
> > +        _tick_label = tick_label
>
> I would do this as
>
> label_ticks_flag = not tick_label is None
>
> Which I think is a bit clearer.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4266/files#r27365250>.
>